src/Elements/ImportMeshElement.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.IO;
+using Elements.Geometry;
+using Elements.Geometry.Interfaces;
+using static Elements.Units;
+
+namespace Elements
+{
+    /// <summary>
+    /// An element definition whose representation is provided by an imported mesh like an STL.
+    /// </summary>
+    /// <example>
+    /// [!code-csharp[Main](../../test/Elements.Tests/ImportMeshElementTests.cs?name=example)]
+    /// </example>
+    [UserElement]
+    public class ImportMeshElement : GeometricElement, ITessellate
```

Comment:
Is it possible now to define a custom element type that wraps a mesh without making changes to `Elements`? If not, is it possible to define a general MeshElement type (e.g. "Model Curve" but for meshes) that can wrap a mesh that may or may not be imported?